.cirrus.yml

```diff
@@ -66,8 +56,8 @@ env:
     ####
     #### Credentials and other secret-sauces, decrypted at runtime when authorized.
     ####
-    # Freenode IRC credentials for posting status messages
-    IRCID: ENCRYPTED[0c4a3cc4ecda08bc47cd3d31592be8ae5c2bd0151bf3def00a9afd139ef1ab23a1bd0523319d076c027f9749ddb1f3c8]
```

Comment:
Did you mean to remove this?

Reply:
Ya, I oinked out all the IRC stuff. It doesn't seem to be used/noticed much.

Reply:
Found some more calls into IRC stuff, and removed.  We should be good now...

Reply:
Is/was that the "[Action Recommended]" podbot messages? Darn. Now that they're not spamflooding the channel, I've actually been checking those - they're a good source of news about flakes. Is there another place where I can get reports of those failures? (Like, is the mailing-list thing working?)

Reply:
Anyhow, if you need to push again (please don't just for this), maybe it might be a good idea to update the commit message as well, to reflect removing the IRC stuff.